fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -261,6 +263,19 @@ public ParseNode visitShowTableStatement(StarRocksParser.ShowTableStatementConte
         }
     }
 
+    @Override
+    public ParseNode visitShowCreateTableStatement(StarRocksParser.ShowCreateTableStatementContext context) {
+        QualifiedName qualifiedName = getQualifiedName(context.qualifiedName());
+        TableName targetTableName = qualifiedNameToTableName(qualifiedName);
+        if (context.MATERIALIZED() != null && context.VIEW() != null) {
+            return new ShowCreateTableStmt(targetTableName, TableType.MATERIALIZED_VIEW);
+        }
+        if (context.VIEW() != null) {
+            return new ShowCreateTableStmt(targetTableName, TableType.VIEW);
+        }
+        return new ShowCreateTableStmt(targetTableName);
```

Comment:
I think for real table, we could add a new type for it, then, view, MV and real table will be unified